src/CONST.js

```diff
@@ -156,6 +156,12 @@ const CONST = {
         NATIVE: 'Native',
     },
 
+    REPORT_ACTION: {
```

Comment:
Hey @dklymenk, I apologize because this is completely my fault for not noticing during my last review, but I just noticed that we also have a `REPORT.ACTIONS.TYPE` constant in this file, so instead of creating this new one we should just put `ADDCOMMENT` there.